skylib/push.bzl

```diff
@@ -93,7 +93,7 @@ def _impl(ctx):
     )
 
     if ctx.attr.image_digest_tag:
-        tag = "$(cat {} | cut -d ':' -f 2 | cut -c 1-7)".format(ctx.outputs.digest.path)
+        tag = "$(cat {} | sed 's/sha256://')".format(image["digest"].path)
```

Comment:
I'm not sure that the proposed change address the root cause of the issue. Could you please provide BUILD file example that reproduce the issue?